src/utils.js

```diff
@@ -147,3 +147,17 @@ export const verifyJWT = (token) => {
     });
   });
 };
+
+/*
+Get JSON Web Token
+*/
+export const getToken = (req, res) => {
```

Comment:
Per my comment above, this function should live in the routes folder as its not really a util because it handles requests and responses